### ISSUE TITLE
Bug in volume_custom_scale_factors

### ISSUE THREAD

**Issue by JaGeo** [autoatml/autoplex#296]
If I use the same scale factor in "volume_custom_scale_factors" multiple times, I will always get the same displaced structure. I haven't looked in detail but I would assume that the structure generation is initialized in the same way over and over again resulting in the same structure if the same seed is used.

**Comment by JaGeo** [comment]
A solution would be to detect if some of the start structures are the same or detecting the similar scaling and then using more than 1 structure in the mc rattle approach: 
https://github.com/autoatml/autoplex/blob/025e8b2cdd6d02d821be287394eb9f8be168791a/src/autoplex/data/common/jobs.py#L307

**Comment by QuantumChemist** [comment]
yeah, it was rather simply implemented at that time, therefore the same factor produces the same structure. I think the same seed is used every time.

**Comment by JaGeo** [comment]
But this might also affect the other data generation. We might need to fix this first before we can restart some of our larger workflows. I fear the resulting structures will be very similar even if the scaling is implemented

**Comment by QuantumChemist** [comment]
Oh, now I understand the consequences of this issue. 🫨
